include/sound/sof/control.h

```diff
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause) */
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
```

Comment:
@plbossart what does GPL-v2.0-only or BSD-3 mean?

Reply:
I have a same question

Reply:
@lyakh @RanderWang The canonical/original GPL license always had the "or later" in license text. Linux kernel specifically omitted this text. So I believe this change is to make it clear the Linux kernel version of the license is referred to here. E.g. https://spdx.org/licenses/GPL-2.0-only.html
I don't believe it has any impact dual-licensing of code, just to specify which version of GPL more exactly.

Reply:
yes, this to clarify that there is no permission to use this code with a GPL-2.0-or-later.
Don't ask me why GPL2.0 can be interpreted in two ways, I am just doing the correction suggested by smarter people than me.

Reply:
@kv2019i @plbossart thanks for the clarification!